src/bin/entities.rs

```diff
@@ -50,8 +51,8 @@ enum Opt {
         topo_id_id: String,
 
         /// Endpoint of the wikibase api
-        #[structopt(short, long, default_value = "http://localhost:8181/api.php")]
-        api: String,
+        #[structopt(short = "a", long = "api")]
+        _api: Option<String>,
```

Comment:
if we won’t use it, can’t we get rid of it completely?

Reply:
I prefer to let it, it makes a unified api between the command (and this eases the tests :stuck_out_tongue_winking_eye: )

Reply:
Ok :+1: